package.yaml

```diff
@@ -88,8 +77,10 @@ executables:
     - -with-rtsopts=-N
     # - -dynamic
     dependencies:
-    - wallet-proxy
+    - mtl
     - optparse-applicative
+    - wallet-proxy
+    - warp
```

Comment:
Why is `warp` needed here explicitly? I don't see any changes in imports in the binary related to this.

Reply:
I moved it from the general dependencies to here, because it is a dependency for the executable, but not for the library.